components/posts/Comments.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState, useEffect } from 'react';
+import { View, Text, FlatList, StyleSheet, TouchableOpacity } from 'react-native';
+import { windowWidth } from '../../utils/Dimensions';
+import Comment from './Comment';
+import { Ionicons } from '@expo/vector-icons';
+import getComments from '../../utils/getComments';
+import PostCommentModal from './PostCommentModal';
+
+const CommentsStyles = StyleSheet.create({
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'flex-start',
+    alignItems: 'flex-start',
+    width: windowWidth * 0.7,
+    marginBottom: 30,
+  },
+  headerContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    alignSelf: 'center',
+    justifyContent: 'space-between',
+    marginBottom: 20,
+    width: '100%'
+  },
+  header: {
+    fontSize: 25,
+    fontWeight: '300',
+  },
+});
+
+const Comments = ({ postID, username }) => {
```

Comment:
It does make sense the way it is laid out, but it is kind of confusing to have Comment.js and Comments.js... It's definitely not a huge deal, but maybe we could rename one of them to make it more clear.